CONTRIBUTING.md

```diff
@@ -158,9 +158,8 @@ in TypeScript.
 
 Changing demos in JavaScript requires a manual update of the TypeScript
```

Comment:
Could we reverse the sentence? "The JavaScript demos are automatically generated. You should only work with the TypeScript demos."

Reply:
You can work on on any version you like. There is no (and must not be) any requirement to work with the TS version.

Reply:
I think that it should be TypeScript only. As soon as a `.tsx` file is present, the CI will fail if you only change the JavaScript demo. We already enforce it.

Reply:
TypeScript is a superset of JavaScript. Enforcing TS only will reduce contributions. Everybody can work on the version they like and collaborators work on making the build green. 

> I think that it should be TypeScript only. As soon as a .tsx file is present

Already a loss of contributions because it's unclear when this should be the case: Do I have to create a TS version if I want to enhance the demo? Should this demo not be changed because it is only in JS? I can't find the TS demo. TS will complain locally. Better not submit the PR etc.

TS is simply not mature enough in the JSX domain to force it. It's a nice to have in our demos that help documentation/testing but a switch will slow us down.

Reply:
We can discuss this in a separate issue. This is not relevant to this PR.

Reply:
Ok, the TypeScript migration effort is pretty new. Let's see how it affects contributions in the coming weeks.